Sorting Algorithms/Merge Sort Algorithm/Java/MergeSort.java

```diff
@@ -0,0 +1,78 @@
+import java.util.Scanner;
+
+/**
+ * Theory of Programming
+ * Created December 25, 2016 by cybersapien
```

Comment:
Please add "http://theoryofprogramming.com/2015/01/29/merge-sort-algorithm/" here and for the sake of consistency, style your comments in this fashion -

```
/* ===== ===== =====

Theory of Programming

<mention_the_algorithm_or_data_structure_name>
<give_the_link_to_corresponding_theory_of_programming_post>
GitHub - https://github.com/VamsiSangam/theoryofprogramming
Code Contributor - <contributor_name>, <other_contributor_name_if_any>

===== ===== ===== */
```